OPAL/br/src/main/scala/org/opalj/br/fpcf/analyses/FieldAccessInformationAnalysis.scala

```diff
@@ -110,14 +110,18 @@ class FieldAccessInformationAnalysis(val project: SomeProject) extends FPCFAnaly
                 fieldAccess._1,
                 FieldAccessInformation.key,
                 (current: EOptionP[Field, FieldAccessInformation]) => current match {
-                    case InterimUBP(ub: FieldAccessInformation) =>
-                        if (readAccesses.subsetOf(ub.readAccesses) && writeAccesses.subsetOf(ub.writeAccesses))
+                    case InterimUBP(ub: FieldAccessInformation) => {
+                        val mergedReadAccesses = ub.readAccesses ++ readAccesses
+                        val mergedWriteAccesses = ub.writeAccesses ++ writeAccesses
+
+                        if (mergedReadAccesses == ub.readAccesses && mergedWriteAccesses == ub.writeAccesses)
```

Comment:
Use identity checks (`mergedReadAccesses eq ub.readAccesses`) instead of equality comparison

Reply:
I am a bit confused about this comment: It was outdated as soon as you wrote it? I pushed my changes before your review, so i have no clue where you even got this code from (as it was removed with i think the latest commit).